README.md

```diff
@@ -1 +1,153 @@
+
+  1. Start grader:
+  ```
+  $ grader -i your_name
+  ```
+  Note: `your_name` is lowercase without whitespace, e.g. `bob`.
+  2. Start the motivation letter grading session, by entering the grading mode:
+  ```
+  grader> grade
+  ```
+  3. This will show you a motivation letter from the set of applicants that you haven't graded already, together with 
+  some additional info
+  4. The prompt will expect you to type one of the following: 
+    - `0, 1, -1` : your grade
+    - `s` : to skip this applicant for now ➔ you will be able to grade it again after restarting the motivation grading in the next grading session
+    - `b` : re-grade the previous motivation letter
+    - `o` : add override (see below for details)
+    - `d` : show the full detail about the current applicant
+    - `l` : add a label to the current applicant (see below for details)
+  5. After pressing enter the next motivation letter will be shown for you to grade
+  6. To quit the grading mode, type `Ctrl+C`.
+  7. By later quitting grader with the command `exit`, your gradings will be stored in the `applications.ini` file. Commit and push to share with the other reviewers. Push often.
+
+Note 1: Unless you type `d`, the data shown is anonymized as far as possible to help you make unbiased decisions.
+
+Note 2: whenever you make changes, also in the following, remember to commit and push.
+
+## Labelling applications
+While in grading mode, you may want to label the current application, for example by setting the `OVERQUALIFIED` label. This marks the applicant as someone who is too much of an *expert* for ASPP. A list of commonly used labels can be found in the `criteria.txt` file.
+
+When you want to add a label to an applicant, you type `l LABEL` at the grading prompt, where `LABEL` is the label you want to add. Note that grader has tab-completion on label names. 
+
+## Use case: changing existing grades
+  - You may want to re-grade a specific applicant. Enter grading mode like this:
+    ```
+    grader> grade Bob Smith
+    ```
+    Note that `grader` has tab-completion on applicant names.
+  - You may want to re-grade/review all the motivation letters you graded with a specific grade, for example `-1`. Enter grading mode like this:
+    ```
+    grader> grade -g -1
+    ```
+ - You may want to re-grade/review all applicants with a specific label:
+   ```
+   grader> grade -l LABEL
+   ```
+
+## Use case: resolve disagreements
+A *disagreement* between reviewer1 and reviewer2 is an applicant for whom the two reviewers gave diverging grades, i.e.`1` and `-1`. After grading all motivations, you may want to go through all disagreements with other reviewers to check if you simply made a typo or if you really disagree. In case of a real disagreement, a discussion among reviewers will be needed. To go through all disagreements with reviewer `reviewer2` enter grading mode like this:
+
+```
+grader> grade -d reviewer2
```

Comment:
Just `grade -d`. This goes through all reviewers.

Maybe add an example how to deal with one reviewer.